src/components/SimpleTable.tsx

```diff
@@ -125,49 +30,108 @@ export interface IProps {
 }
 
 // TODO generalize this component - currently tailored for stablecoin market caps
-const SimpleTable: React.FC<IProps> = ({ columns, content, hasError }) => (
-  <Table>
-    <Header>
-      {columns.map((column, idx) => (
-        <HeaderCell key={idx}>{column}</HeaderCell>
-      ))}
-    </Header>
-    {hasError && (
-      <Row>
-        <Cell>
-          <Translation id="page-stablecoins-table-error" />
-        </Cell>
-      </Row>
-    )}
-    {!hasError && content.length === 0 && (
-      <Row>
-        <Cell>
-          <Translation id="page-stablecoins-table-loading" />
-        </Cell>
-      </Row>
-    )}
-    {content.map((row, idx) => {
-      const { name, marketCap, image, type, url } = row
-      const rowContent = (
-        <>
-          <Cell>
-            {image && <StyledImage src={image} alt="" />}
-            {name}
-          </Cell>
-          <Cell>{marketCap}</Cell>
-          <Cell>{type}</Cell>
-        </>
-      )
+const SimpleTable: React.FC<IProps> = ({ columns, content, hasError }) => {
+  const [textColor] = useToken("colors", ["text"])
+
+  return (
+    <Table
+      variant="unstyled"
+      my={8}
+      borderRadius="2px"
+      border={`1px solid ${textColor}`}
+      bg="background"
+      mb={8}
+      minW="720px"
+    >
+      <Thead bg="ednBackground" color="text200">
+        <Tr boxShadow={`0 1px 1px ${textColor}`} mb="1px">
+          {columns.map((column, idx) => (
+            <Th
+              key={idx}
+              py={5}
+              fontSize="md"
+              fontWeight="normal"
+              letterSpacing="normal"
+            >
+              {column}
+            </Th>
+          ))}
 
-      return url ? (
-        <LinkRow key={idx} to={url}>
-          {rowContent}
-        </LinkRow>
-      ) : (
-        <Row key={idx}>{rowContent}</Row>
-      )
-    })}
-  </Table>
-)
+          {content && content[0]?.url && (
+            <Th p={5} fontSize="md" fontWeight="normal" textAlign="end">
+              ↗
+            </Th>
+          )}
+        </Tr>
+      </Thead>
+      <Tbody>
+        {hasError && (
+          <Tr p={4}>
+            <Td colSpan={4}>
+              <Translation id="page-stablecoins-table-error" />
+            </Td>
+          </Tr>
+        )}
+        {!hasError && content.length === 0 && (
+          <Tr p={4}>
+            <Td colSpan={4}>
+              <Translation id="page-stablecoins-table-loading" />
+            </Td>
+          </Tr>
+        )}
 
+        {content.map(({ name, marketCap, image, type, url }, idx) => (
+          <Tr
+            as={url ? LinkBox : undefined}
```

Comment:
@amit-ksh I think we need to do the inverse here.

```suggestion
          <LinkBox
            as={url ? Tr : undefined}
```

in order to render a `tr` element instead of a `div` (which is what the `LinkBox` renders).

![image](https://user-images.githubusercontent.com/468158/209213596-4bb2a497-d55e-4a58-a4ad-f195d7852c1f.png)

I think that would fix the issue, right?

cc @TylerAPfledderer

Reply:
In this case, `div` will appear if, the `url` is `undefined`
I think we should use like this  
```jsx
<LinkBox
    as={Tr}
```
This will definitely render `tr` and also work as a `LinkBox`

Reply:
Correct! good catch. Yeah I think we should always render a `Tr` here. Then if the `url` does not exist we don't show the `LinkOverlay` (something we are already doing) 👍🏼

Reply:
Yes this is fine, because `LinkBox` only really renders a `position: relative` which is used for `LinkOverlay`. :)